### PR TITLE
PyTorch notebook 2.1 draft

### PR DESCRIPTION
**Pull request by xiejinglei**
A first draft of PyTorch notebook 2.1 has been created.

### CODE REVIEW

**Comment by xiejinglei** [comment]
Some issues may need to be considered:

1. Whenever I do `fit` or `evaluate`, there will be a warning like the following:
    ```scala
    Warn: jep.JepException: <class 'StopIteration'>
            at jep.Jep.exec(Native Method)
            at jep.Jep.exec(Jep.java:478)
            at com.intel.analytics.zoo.common.PythonInterpreter$$anonfun$1.apply$mcV$sp(PythonInterpreter.scala:108)
            at com.intel.analytics.zoo.common.PythonInterpreter$$anonfun$1.apply(PythonInterpreter.scala:107)
            at com.intel.analytics.zoo.common.PythonInterpreter$$anonfun$1.apply(PythonInterpreter.scala:107)
            at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
            at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
            at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
            at java.util.concurrent.FutureTask.run(FutureTask.java:266)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
            at java.lang.Thread.run(Thread.java:748)
    ```
    For now I just remove the message manually, since I haven't find a way to solve this warning.
2. The notebook is currently placed in new folder `pytorch`. Maybe it needs to be changed to keep consistency with other notebooks.

File: pytorch/2.1-a-first-look-at-a-neural-network.ipynb [hunk 1]
```diff
@@ -0,0 +1,547 @@
...
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Trained 320.0 records in 0.052754554 seconds. Throughput is 6065.827 records/second. Loss is 0.07037376. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:472 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Epoch finished. Wall clock time is 87921.741495 ms\n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:111 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Validate model...\n",
+      "[Stage 1893:>                                                       (0 + 1) / 1]\n",
+      "2021-02-22 17:31:51 INFO  DistriOptimizer$:177 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] validate model throughput is 5835.1914 records/second\n",
+      "2021-02-22 17:31:51 INFO  DistriOptimizer$:180 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Top1Accuracy is Accuracy(correct: 9791, count: 10000, accuracy: 0.9791)\n"
```
Comment (xiejinglei) [inline pytorch/2.1-a-first-look-at-a-neural-network.ipynb:413]: The validation accuracy is already displayed here. Is it still necessary to do `est.evaluate(data=test_loader, batch_size=test_batch_size)` ?

Comment (hkvision) [reply pytorch/2.1-a-first-look-at-a-neural-network.ipynb:412]: We can put it here to demonstrate how to do evaluation.

**Comment by hkvision** [comment]
It looks good.
